src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -15,6 +15,10 @@ public static class Utilities
         public static readonly string PreloaderAssembly = "LibG.AsmPreloader.Managed.dll";
         public static readonly string PreloaderClassName = "Autodesk.LibG.AsmPreloader";
         public static readonly string PreloaderMethodName = "PreloadAsmLibraries";
+        // Key words for Products containing ASM binaries
+        private static readonly List<string> ProductsWithASM = new List<string>() { "Revit", "Civil", "FormIt" };
```

Comment:
So these product names are the same as they appear in the registry? Does the order of product names matter in terms of choosing an ASM installation to use?